torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -826,7 +827,21 @@ void LLVMCodeGenImpl::visit(const Cast* v) {
       } else {
         value_ = irb_.CreateSIToFP(value_, dstType);
       }
+
     } else if (dstType->isIntOrIntVectorTy()) {
+      // Bool is represented as 8bits, we need to make sure
+      // we're appropriately setting the one bit that is used
+      if (v->dtype().scalar_type() == ScalarType::Bool) {
+        llvm::Constant* constant_zero = llvm::ConstantInt::get(srcType, 0);
+        if (v->src_value()->dtype().lanes() != 1) {
+          std::vector<llvm::Constant*> const_vector;
```

Comment:
I think `zeroes` instead of `const_vector` and `zero` instead of `constant_zero` would be both shorter and more descriptive. Knowing it's a zero already implies it's a constant.